metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cirrax-libvirt",
-  "version": "2.1.3",
+  "version": "2.1.4",
   "author": "Cirrax GmbH",
```

Comment:
can you please leave the version number as is. I release versions with blacksmith scripts, so increasing here does not work well with my workflow...